sound/hybrid_font.h

```diff
@@ -7,10 +7,14 @@ class IgniterConfigFile : public ConfigFile {
     CONFIG_VARIABLE(humStart, 100);
     CONFIG_VARIABLE(volHum, 15);
     CONFIG_VARIABLE(volEff, 16);
+    CONFIG_VARIABLE(clashLockOn, 1);
```

Comment:
When does it ever make sense to play a clash before lockon?
Clash sounds usually have some sort of fade-out or echo which makes that unsteasonable.
And if you really wanted this, why not just copy the clash sounds to lockon*.wav?

Reply:
that also sounds reasonable. why not remove that then for the main repo? that would be fine!